### DISCUSSION TITLE
How to handle sites stream video in multiple files?

### DISCUSSION THREAD

**Discussion by micnolmad** [aclap-dev/video-downloadhelper#1573] [Help and Q&A]
### Your question
Update
So this only happens when the movie is paused. If left running, the stream is in one file. Now the only problem is that the hole thing looks like back in 90s with satellite services encrypted the stream. Closing this topic.

----------------
I am trying to download a movie that we have on blu ray but the language is wrong and only the streaming services have it in our own language. So in reality I really only need the audio but I think I have to demux it from the full mp4 as those are the only thing I see in network tap in FF. But there are so many mp4 files of 32kb size that I don't know if vdh can even handle this case.

Back when I used AHDL or somwthing like that, it would stich all the little files together but that was from public streaming services and didn't have to deal with login and drm.

Can vdh handle these sites?

When I open vdh I see a long list of mp4 files just as I do in inspect mode in FF. Where do I go from here?

Sorry for website obfuscation but since this is all public I figured maybe this will help keep it less likely to hit a search hit at that site.

~micnolmad

### Addon & CoApp version and details

vdh 9.1.0.44
coapp v2.0.19
ff 133.0.3 (64-bit)
windows 10 64bit
source https://www. _**WEBSITE**_ .com/watch/asset/movies/sonic-the-hedgehog/
website s.k.y-s-h-o-w,t,i,m,e

**Comment by Mattmjs** [comment]
For your information since the topic is closed, it sounds like the site in question uses DRM. You can verify this in Firefox if you see this icon.
Any site using DRM will show this.

![DRM icon](https://github.com/user-attachments/assets/476d24ba-dcca-4d0e-9ece-c52dbc72656a)

**Reply by micnolmad** [reply]
It is encrypted. I thought at the time that the MP4 was in decrypted state. I guess it happens on the fly. Thank you for chiming in and merry Christmas 🎄
